presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -279,10 +280,20 @@ private void alterTable(String databaseName, String tableName, org.apache.hadoop
         if (partitionNames.isEmpty()) {
             return ImmutableMap.of();
         }
+
+        Function<org.apache.hadoop.hive.metastore.api.Partition, Partition> fromMetastoreApiPartition = ThriftMetastoreUtil::fromMetastoreApiPartition;
+        boolean isAvroTableWithSchemaSet = delegate.getTable(databaseName, tableName)
+                .map(ThriftMetastoreUtil::isAvroTableWithSchemaSet)
+                .orElse(false);
+        if (isAvroTableWithSchemaSet) {
+            List<FieldSchema> schema = delegate.getFields(databaseName, tableName).get();
+            fromMetastoreApiPartition = partition -> fromMetastoreApiPartition(partition, schema);
```

Comment:
The static import here makes this confusing, since the name is the same as the variable. Let's remove the static import.

Reply:
Yeah, this looks ambiguous on Github. Fortunately, in IDE this gets colored clearly.